### PR TITLE
update install instructions for mac; fix next steps

### PR DESCRIPTION
**Pull request by aaronmarkham**
*Issue #, if available:*
Somewhat addresses #120 

*Description of changes:*
Next steps referred to the old beta site which isn't maintained. Also, there were no callouts to the tutorials or API in this project, so I added those.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhanghang1989** [comment]
close this pr in favor of PR https://github.com/awslabs/autogluon/pulls

**Comment by aaronmarkham** [comment]
> close this pr in favor of PR https://github.com/awslabs/autogluon/pulls

You didn't mention the specific PR you're using instead.
Did you also patch the next steps?

File: docs/install.rst [hunk 1]
```diff
@@ -12,12 +12,13 @@ Installation
 Next steps
 ~~~~~~~~~~
 
-- Checkout `beta.mxnet.io <http://beta.mxnet.io/install/index.html>`_ for more options such as ARM devices and docker images.
-- `Verify your MXNet installation <https://beta.mxnet.io/install/validate-mxnet.html>`_
+- `AutoGluon tutorials </tutorials/index.html>`_
```
[the remark this answers is not in the document]
Comment (aaronmarkham) [reply docs/install.rst]: It has next steps that are more appropriate for another repo... probably copied. So if you have those, you should lead with the ones directly related to this repo.

**Comment by zhanghang1989** [comment]
> > close this pr in favor of PR https://github.com/awslabs/autogluon/pulls
> 
> You didn't mention the specific PR you're using instead.
> Did you also patch the next steps?

My bad. It is in Jonas' PR https://github.com/awslabs/autogluon/pull/133

**Comment by aaronmarkham** [comment]
> > > close this pr in favor of PR https://github.com/awslabs/autogluon/pulls
> > 
> > 
> > You didn't mention the specific PR you're using instead.
> > Did you also patch the next steps?
> 
> My bad. It is in Jonas' PR #133

Ya, that doesn't include the changes I made to the install next steps. If you reopen this, I'll remove the mac stuff that isn't relevant anymore.

**Comment by zhanghang1989** [comment]
@aaronmarkham so sorry about this. I thought you are not updating this PR. Could you reopen a new one, because I didn't see the Reopen button in this page.

**Comment by aaronmarkham** [comment]
> @aaronmarkham so sorry about this. I thought you are not updating this PR. Could you reopen a new one, because I didn't see the Reopen button in this page.

I think because it went public, and mine was private until just now, I still can't make a new PR to fix this. It'll have to wait.

**Comment by zhanghang1989** [comment]
> > @aaronmarkham so sorry about this. I thought you are not updating this PR. Could you reopen a new one, because I didn't see the Reopen button in this page.
> 
> I think because it went public, and mine was private until just now, I still can't make a new PR to fix this. It'll have to wait.

I am having the same issue. We may need re-fork the repo.

**Comment by aaronmarkham** [comment]
> > > @aaronmarkham so sorry about this. I thought you are not updating this PR. Could you reopen a new one, because I didn't see the Reopen button in this page.
> > 
> > 
> > I think because it went public, and mine was private until just now, I still can't make a new PR to fix this. It'll have to wait.
> 
> I am having the same issue. We may need re-fork the repo.

I had to delete my old fork, and fork again.

New PR: #143

**Pull request closed without merging**